components/lv-ts-tableprovider/TSTableProviders/pom.xml

```diff
@@ -0,0 +1,36 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.tibco.ep.ldm.tstables</groupId>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>LV technical support tables</name>
+  <description>TableProvider classes to add TS of LiveView</description>
+  
+    <properties>
+    	<com.tibco.ep.sb.studio.sample>true</com.tibco.ep.sb.studio.sample>

```

Comment:
the com.tibco.ep.studio.sample property should not be defined here

Reply:
OK, I removed from both sub-projects.